packaging/pyinstaller/libertem-server.spec

```diff
@@ -0,0 +1,68 @@
+# -*- mode: python ; coding: utf-8 -*-
+
+import sys ; sys.setrecursionlimit(sys.getrecursionlimit() * 5)
+
+dask_datas = []
+
+dask_files = [
+    ('Lib/site-packages/dask/dask.yaml', 'dask'),
```

Comment:
This should be unnecessary, as [existing hook for dask](https://github.com/pyinstaller/pyinstaller-hooks-contrib/blob/master/src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-dask.py) should collect that file.